branchpro/tests/test_models.py

```diff
@@ -20,3 +21,35 @@ def test_simulate(self):
         forward_model = bp.ForwardModel()
         with self.assertRaises(NotImplementedError):
             forward_model.simulate(0, 1)
+
+
+class TestBrachProModelClass(unittest.TestCase):
+    """
+    Test the 'BranchProModel' class.
+    """
+    def test__init__(self):
+        branch_model = bp.BranchProModel(0, [1])
+        self.assertEqual(branch_model._serial_interval, np.array([1]))
```

Comment:
Could you please remove those asserts of the private properties? This is again according to the philosophy that private things are not tested